src/model/remoteConfig.ts

```diff
@@ -226,10 +226,11 @@ export async function fetchRemoteConfig(): Promise<RainbowConfig> {
     });
     return config;
   } catch (e) {
+    if (e instanceof Error && e.message === 'The request timed out') return DEFAULT_CONFIG;
     logger.error(new RainbowError(`[remoteConfig]: Failed to fetch remote config`), {
       error: e,
     });
-    throw e;
+    return DEFAULT_CONFIG;
```

Comment:
bubbling up from this fetch call wasn't wrapped in a try/catch so we were seeing the error boundary here from this

Reply:
biggest offender was timeout for `fetchAndActivity()`

Reply:
Should this instead return `getRemoteConfig()`? In case there's newer data than what's in `DEFAULT_CONFIG`.